modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1157,6 +1157,30 @@ impl MintClientModule {
         self.get_note_counts_by_denomination(dbtx).await
     }
 
+    /// Estimates the total fees to spend all currently held notes.
+    ///
+    /// This is useful for calculating max withdrawable amounts, where all
+    /// notes will be spent. Notes that are uneconomical to spend (fee >= value)
+    /// are excluded from the calculation since the wallet won't spend them.
```

Comment:
Thinking out loud: why would the mint ever allow us to have a note that is "uneconomical"?